packages/artifact/src/internal/download-http-client.ts

```diff
@@ -263,26 +263,44 @@ export class DownloadHttpClient {
       if (isGzip) {
         const gunzip = zlib.createGunzip()
         response.message
+          .on('error', error => {
```

Comment:
We could monitor for `aborted` as well. https://nodejs.org/dist/latest-v14.x/docs/api/http.html#http_class_http_incomingmessage